adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -84,7 +84,6 @@ void HybrisProximityAdaptor::sendInitialData()
 
        struct input_absinfo absinfo;
        int fd;
-       inputDev.replace("input","event");
        inputDev.prepend("/dev/input/");
 
        if ((fd = open(inputDev.toLatin1(), O_RDONLY)) > 0) {
```

Comment:
This looks bad: While open() returning 0 is really unlikely,  it would still be a success.